charts/latest/spdk-csi/templates/storage-node.yaml

```diff
@@ -101,7 +101,7 @@ spec:
       - name: s-node-api-container
         image: "{{ $.Values.image.simplyblock.repository }}:{{ $.Values.image.simplyblock.tag }}"
         imagePullPolicy: "{{ $.Values.image.simplyblock.pullPolicy }}"
-        command: ["python", "simplyblock_web/snode_app_k8s.py"]
+        command: ["python", "simplyblock_web/node_webapp.py", "storage_node_k8s"]
```

Comment:
@Hamdy-khader I don't see change for this in `sbcli` side

Reply:
@geoffrey1330 it's here: https://github.com/simplyblock-io/sbcli/pull/237/files - needs to be merged before this one

Reply:
it is in this PR,
https://github.com/simplyblock-io/sbcli/pull/237/files